src/app/frontend/deploy/deploy_controller.js

```diff
@@ -43,10 +41,19 @@ export default class DeployController {
     this.namespaces = namespaces.namespaces;
```

Comment:
I wanted to do so, but there is no resolve function for directives. So I left list of namespaces here. I do see, however, a room for improvement. For example, we could use Angular 1.5 activate method for directives. Until then, moving it to directive initialization phase would make things more complicated, I think